README.md

```diff
@@ -58,6 +58,13 @@ We thank everyone who supported us financially in the past and special thanks to
 
 3. Sync gradle and rebuild your app
 
+*Note: MapLibre by default ships with the proprietary Google Play Location Services. If you want to avoid pulling proprietary dependencies into your project, you can exclude Google Play Location Services as follows:*
+```gradle
+    implementation ('org.maplibre.gl:android-sdk:9.4.0') {
```

Comment:
```suggestion
    implementation ('org.maplibre.gl:android-sdk:<version>') {
```

Reply:
Like that we don't have to update the readme at every version...

Reply:
__

Reply:
I guess you want that changed on line 54 then too?

Reply:
Yes good catch. Can you update it?